autobid.py

```diff
@@ -1,15 +1,14 @@
 #!/usr/bin/python3
-
 from __future__ import absolute_import, print_function
-
 import argparse
 import pprint
 import sys
 import traceback
 from time import sleep
-
 import config
 from logic import bidding_logic, epoch_logic, output_logic, validator_logic
+#vstatsAlertInit
+from vstats.alerts import *
```

Comment:
Can you import each module by name instead of *?

https://www.geeksforgeeks.org/why-import-star-in-python-is-a-bad-idea/#:~:text=Using%20import%20*%20in%20python%20programs,other%20libraries%20that%20you%20import.